tauri/src/api.rs

```diff
@@ -16,27 +18,20 @@ pub fn handler<T: 'static>(webview: &mut WebView<'_, T>, arg: &str) -> bool {
               _webview
                 .eval(&format!(
                   "window['{queue}'] = [];
-                  window['{fn}'] = function (payload, salt, ignoreQueue) {{
-                    window.tauri.promisified({{
-                      cmd: 'validateSalt',
-                      salt: salt
-                    }}).then(function () {{
```

Comment:
Will this be coming back?

Reply:
yeah, and we also have to think about locking every API call with salt